server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -3183,7 +3184,7 @@ public DiskOffering updateDiskOffering(final UpdateDiskOfferingCmd cmd) {
             throw new InvalidParameterValueException(String.format("Unable to update disk offering: %s by id user: %s because it is not root-admin or domain-admin", diskOfferingHandle.getUuid(), user.getUuid()));
         }
 
-        final boolean updateNeeded = name != null || displayText != null || sortKey != null || displayDiskOffering != null;
+        final boolean updateNeeded = isUpdateDiskOfferingNeeded(name, displayText, sortKey, displayDiskOffering, tags);
```

Comment:
`shouldUpdateDiskOffering` sounds better to me. However, I am not a native speaker. Therefore, change only if you also agree with me.